x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/integration/MlDistributedFailureIT.java

```diff
@@ -183,6 +183,7 @@ public void testCloseUnassignedJobAndDatafeed() throws Exception {
     }
 
     @TestLogging("org.elasticsearch.xpack.ml.action:TRACE,org.elasticsearch.xpack.ml.process:TRACE")
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/pull/36069")
```

Comment:
Will this be removed in the next PR?

Reply:
No, I'll get that working before merging this PR.